client/client.go

```diff
@@ -41,12 +42,12 @@ var (
 	InvalidDCInURL     = "invalid data center in URL"
 
 	knownDCFormats = []string{
-		`https?://(.*).api.joyent.com`,
-		`https?://(.*).api.joyentcloud.com`,
 		`https?://(.*).api.samsungcloud.io`,
+		`https?://(.*).api.tritoncompute.cloud`,
+		`https?://(.*).api.mnx.io`,
```

Comment:
Remove samsung and tritoncompute.cloud. Honestly, the whole "short hand dc reference" should be eliminated entirely.

Reply:
This has been fixed. See comment below regarding removal of "short hand dc reference".